src/main/java/com/squareup/kotlinpoet/FunSpec.kt

```diff
@@ -143,7 +161,7 @@ class FunSpec private constructor(builder: Builder) {
   override fun hashCode() = toString().hashCode()
 
   override fun toString() = buildString {
-    emit(CodeWriter(this), "Constructor", TypeSpec.Kind.Class().implicitFunctionModifiers)
+    emit(CodeWriter(this), "Constructor", TypeSpec.Kind.Class().implicitFunctionModifiers, true)
```

Comment:
Would be great to use named args for boolean values

Reply:
Added arg names, thanks.